index.ts

```diff
@@ -21,29 +21,28 @@ export function* sinergia(
       const itemIterator: IterableIterator<any> = task(accumulator, item);
 
       yield new Promise(resolve => {
-        const step = (timestamp) => { // timestamp not used
+        const step = () => {
           const iteration = itemIterator.next();
 
-          if (!iteration.done) {
-            window.requestAnimationFrame(step);
-          }
-          else {
+          if (iteration.done) {
```

Comment:
I like ESLint rule about negations but I find personally don't find this case hard to understand. Moreover it feels more natural having `if (!iteration.done)` first because it's the block which is ran unless the last task is completed.